backend/app/api/src/helpers/SetupStepsUpdater.ts

```diff
@@ -33,6 +39,9 @@ export class SetupStepUpdater {
             if(premiseTypes) {
                 platform.config.premiseTypes = premiseTypes;
             }
+            if(responsibilities) {
+                platform.config.responsibilities = responsibilities;
+            }
```

Comment:
Zou dit niet mogelijk maken, de aanpassingen moeten opgeslagen zijn voor we de steps herbekenen, anders krijgen we onvoorspelbare race conditions